etc/behat/services/pages.xml

```diff
@@ -33,6 +36,11 @@
         <parameter key="sylius.behat.page.external.paypal_checkout_express.class">Sylius\Behat\Page\External\PaypalExpressCheckoutPage</parameter>
     </parameters>
     <services>
+        <service id="sylius.behat.page.admin.product_show" class="%sylius.behat.page.admin.product_show.class%" parent="sylius.behat.symfony_page" scope="scenario" public="false" />
```

Comment:
Shouldn't be better to base this classes on @Arminek work from #4395 ? Just a thought.